### ISSUE TITLE
[Desktop Build Windows Store] The language choice has font problem

### ISSUE THREAD

**Issue by RabloGames** [Scirra/Construct-3-bugs#1807]
## Problem description

On the Windows (not NW.js) desktop build, the language selection is buggy. The font doesn't take special characters.

## Attach a .c3p

Screenshot of the problem:
https://i.imgur.com/KZfjM5I.jpg

## Steps to reproduce

1. Download the Windows Store desktop build and go to parameters
2. All languages names except English look buggy.

## Observed result

The font is buggy for all languages but English.

## Expected result

The languages names should be correctly displayed, as in the other versions of C3.

## Affected browsers

Desktop build (Windows Store only)

## System details

<details><summary>View details</summary>

Platform information
Browser: Edge
Browser version: 17.17134
Browser engine: Edge
Browser architecture: 64-bit
Context: windows-store
Operating system: Windows
Operating system version: 10
Operating system architecture: 64-bit
Device type: desktop
Device pixel ratio: 1
Logical CPU cores: 8
Approx. device memory: (unavailable)
User agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; MSAppHost/3.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36 Edge/17.17134
C3 release: r109 (beta)
Language setting: fr-FR 
Local storage
Storage quota (approx): (status unavailable)
Storage usage (approx): (status unavailable)
Persistant storage: (status unavailable) 
Browser support notes
This list contains missing features that are not required, but could improve performance or user experience if supported.
CSS containment is not supported. Editor performance may be significantly degraded.
The <dialog> element is not supported. A polyfill is in use.
The Clipboard API is not supported. Some clipboard features may be unavailable.
Web Animations are not supported. Animations are disabled.
This browser does not support pluralisation rules. Plural forms may not be correct.
WebGL 2+ is not supported. Rendering quality and features may be affected.
ImageBitmap is not supported. Texture loading performance may be degraded.
Idle callbacks are not supported. Background loading performance may be degraded.
Canvas toBlob is not supported. Image processing performance may be degraded.
Determining input device capabilities is not supported.
image-set() is not supported. Mouse cursors may appear with reduced quality.
WebGL information
Version string: WebGL 1.0
Numeric version: 1
Supports NPOT textures: partial
Supports GPU profiling: no
Vendor: Microsoft
Renderer: ANGLE (NVIDIA GeForce GTX 960 Direct3D11 vs_5_0 ps_5_0)
Major performance caveat: no
Maximum texture size: 16384
Point size range: 1 to 1024
Extensions:
WEBGL_compressed_texture_s3tc
OES_texture_float
OES_texture_float_linear
EXT_texture_filter_anisotropic
OES_standard_derivatives
ANGLE_instanced_arrays
OES_element_index_uint
WEBGL_debug_renderer_info
WEBGL_depth_texture
EXT_frag_depth
OES_texture_half_float
OES_texture_half_float_linear
WEBGL_lose_context
OES_vertex_array_object
WEBGL_draw_buffers
EXT_blend_minmax
EXT_shader_texture_lod
EXT_color_buffer_half_float
WEBGL_color_buffer_float
Audio information
System sample rate: 48000 Hz
Output channels: 2
Output interpretation: speakers
Supported decode formats:
Ogg Opus (audio/ogg; codecs=opus)
WebM Opus (audio/webm; codecs=opus)
Ogg Vorbis (audio/ogg; codecs=vorbis)
WebM Vorbis (audio/webm; codecs=vorbis)
MPEG-4 AAC (audio/mp4; codecs=mp4a.40.5)
MP3 (audio/mpeg)
FLAC (audio/flac)
PCM WAV (audio/wav; codecs=1)
Supported encode formats: (encoding not supported)
Video information
Supported decode formats:
WebM VP9 (video/webm; codecs=vp9)
WebM VP8 (video/webm; codecs=vp8)
Ogg Theora (video/ogg; codecs=theora)
H.265 (video/mp4; codecs=hev1.1.2.L93.B0)
H.264 (video/mp4; codecs=avc1.42E01E)
Supported encode formats: (encoding not supported)

</details>

**Comment by AshleyScirra** [comment]
Looks like it does the same in Edge so not specific to the desktop app

**Comment by AshleyScirra** [comment]
This is weird, it looks like Edge gets the wrong text encoding for the HTML, but only in production. Development copy appears OK. I've made a change to try to get the right encoding for the next release, we'll have to check it when it's out to see if it worked though.

**Comment by AshleyScirra** [comment]
Edge ignored yet another "use UTF-8" directive and still seems to garble the list... sigh... we'll have to try something else!

**Comment by AshleyScirra** [comment]
Put in another way to force UTF-8 encoding, let's see if that works in the next release...

**Comment by AshleyScirra** [comment]
Nope! This is really annoying. I've come up with yet another way to coax Edge in to using UTF-8 encoding, lets see if *that* works in the next release...
